assets/css/global.less

```diff
@@ -1544,6 +1545,41 @@ body .guides .guides-section-white {
   }
 }
 
+.no-azure-results{
+  h2 {
+    color: @guides-font;
+    font-size: 30px;
+    line-height: normal;
+    letter-spacing: 0.84px;
+    margin-bottom: 8px;
+    margin-top: 91px;
+    &:hover > a.anchorjs-link{
+      color: #FFFFFF;
+      text-decoration: none;
+      display: none;
+    }
+  }
+  p {
+    font-size: 20px;
+    opacity: 0.7;
+    line-height: 1.5;
+    letter-spacing: 0.56px;
+    color: #000000;
+    margin-bottom: 24px;
```

Comment:
Does this part of the css still need to exist?

Reply:
Yes it does

Reply:
<img width="451" alt="Screenshot 2019-09-17 at 4 22 28 PM" src="https://user-images.githubusercontent.com/26963758/65055598-62906580-d967-11e9-8a5f-29fe6a39650c.png">

Reply:
Oh ok! I was confused because of the opacity, I thought this was about that old background